API/front.py

```diff
@@ -0,0 +1,27 @@
+from flask import render_template, Blueprint
+import requests
+import json
+from datetime import datetime
+
+front = Blueprint('front', __name__, template_folder='templates')
+@front.route('/')
+def template_test():
+    # We get the user information from our mocked API
+    user = requests.get('https://hackathoooon.osc-fr1.scalingo.io/mock/pass/calendar?user=1')
+    user_calendar = json.loads(user.text)['calendar']
+
+    # We search the next_course
+    today = datetime.today()
+    today_course = ""
```

Comment:
Si tu trouve pas de cours pour aujourd'hui, `today_course` sera toujours un string vide quand tu vas faire le rendu de ton template, et ça va planter.
```suggestion
    today_course = {}
```